liability.qmd

```diff
@@ -8,12 +8,60 @@ title: "Liability with OS"
+
+Most open source licenses include a disclaimer of liability and warranty. So according the license there is no liability and warrenty from the source code provider.
+
+Examples:
+
+-   [MIT License](https://opensource.org/licenses/MIT){target="_blank"}
+
+    >The software is provided “as is”, **without warranty** of any kind, express or implied, including but not limited to the warranties of merchantability, fitness for a particular purpose and noninfringement. In **no event shall** the authors or copyright holders **be liable** for any claim, damages or other liability, whether in an action of contract, tort or otherwise, arising from, out of or in connection with the software or the use or other dealings in the software.
+
+-   [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0.html){target="_blank"}
+
+    >7. Disclaimer of Warranty <br/>
+    Unless required by applicable law or agreed to in writing, Licensor provides the Work (and each Contributor provides its Contributions) on an “AS IS” BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied, including, without limitation, any warranties or conditions of TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A PARTICULAR PURPOSE. You are solely responsible for determining the appropriateness of using or redistributing the Work and assume any risks associated with Your exercise of permissions under this License.<br/><br/>
+    >8. Limitation of Liability<br/>
+    In no event and under no legal theory, whether in tort (including negligence), contract, or otherwise, unless required by applicable law (such as deliberate and grossly negligent acts) or agreed to in writing, shall any Contributor be liable to You for damages, including any direct, indirect, special, incidental, or consequential damages of any character arising as a result of this License or out of the use or inability to use the Work (including but not limited to damages for loss of goodwill, work stoppage, computer failure or malfunction, or any and all other commercial damages or losses), even if such Contributor has been advised of the possibility of such damages.<br/><br/>    
+    >9. Accepting Warranty or Additional Liability<br/>
+    While redistributing the Work or Derivative Works thereof, You may choose to offer, and charge a fee for, acceptance of support, warranty, indemnity, or other liability obligations and/or rights consistent with this License. However, in accepting such obligations, You may act only on Your own behalf and on Your sole responsibility, not on behalf of any other Contributor, and only if You agree to indemnify, defend, and hold each Contributor harmless for any liability incurred by, or claims asserted against, such Contributor by reason of your accepting any such warranty or additional liability.
+
+-   [GNU General Public License v3.0](https://www.gnu.org/licenses/gpl-3.0.html){target="_blank"}
+
+    >15. Disclaimer of warranty.<br/>
```

Comment:
I did not cross-check the bullet numbering with the source doc - but I presume that they are consistent (e.g., "Disclaimer of warranty" is bullet 15 in the GNU v3.0). If you've checked, I'm good with it. No QA needed!